internal/test/framework/windowsvm.go

```diff
@@ -64,6 +67,11 @@ type WindowsVM interface {
 	Reinitialize() error
 	// Destroy destroys the Windows VM
 	Destroy() error
+	// BuildWMCB returns the value of buildWMCB. It can be used by WSU to decide if it should build WMCB before using it
+	BuildWMCB() bool
```

Comment:
Because https://golang.org/doc/effective_go.html#Getters lists 

> it's neither idiomatic nor necessary to put Get into the getter's name. If you have a field called owner (lower case, unexported), the getter method should be called Owner (upper case, exported), not GetOwner. The use of upper-case names for export provides the hook to discriminate the field from the method.